crates/hc_service_check/CHANGELOG.md

```diff
@@ -7,4 +7,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## 0.1.0-beta-rc.0
```

Comment:
I'm pretty sure we intended to publish this but that won't happen when there is no previous heading